src/player.js

```diff
@@ -437,6 +444,7 @@ export default class Player extends FakeEventTarget {
     this._createPlayerContainer();
     this._appendDomElements();
     this._externalCaptionsHandler = new ExternalCaptionsHandler(this);
+    this._externalThumbnailsHandler = new ExternalThumbnailsHandler(this);
```

Comment:
pass just the relevant configuration directly to the load() and not save state at all
and make sure reset - reset the entire state